### ISSUE TITLE
Execute callback even if visibility didn't change

### ISSUE THREAD

**Issue by onemanclapping** [thenikso/angular-inview#115]
`// Don't forward if no relevant infomation changed
        if (!info.changed) {
          return false;
        }`

Updating from 1.x to 2.x I noticed this, which kinda ruined a use case of mine. [I basically have a growing list with a spinner at the end.](https://github.com/onemanclapping/ng-lazy-render/blob/master/src/lazy-repeater-directive.js) When the spinner is visible, the content is rendered but the spinner element is kept for further growing the list when the user scrolls some more.

Problem: if the initial value of the list sub-set is small, the spinner is immediately seen and more data rendered. If the spinner is still visible (big screens), nothing happens because effectively nothing changed in terms of visibility, but I want the callback to be called anyway.

Could you consider making this optional? I can make a PR if you agree. Thanks!

**Comment by thenikso** [comment]
Yes I think that having it optional makes a lot of sense. Especially if it breaks use cases with 1.0. PRs are very welcome!
